src/js/va-letters/containers/VALettersApp.jsx

```diff
@@ -1,19 +1,60 @@
+import PropTypes from 'prop-types';
 import React from 'react';
 import { connect } from 'react-redux';
 
+import RequiredLoginView from '../../common/components/RequiredLoginView';
+
+// This needs to be a React component for RequiredLoginView to pass down
+// the isDataAvailable prop, which is only passed on failure.
+function AppContent({ children, isDataAvailable }) {
+  const unregistered = isDataAvailable === false;
+  let view;
+
+  if (unregistered) {
+    view = (
+      <h4>
+        To view and download your VA letters, you need something something something.
+      </h4>
+    );
+  } else {
+    view = children;
+  }
+
+  return (
+    // TODO: add HTML and CSS when design has settled
+    <div className="row">
```

Comment:
My PR will eventually change some of this, but I'm trying not to use new Foundation class names so we don't add to the problem of stripping out Foundation :). You should be able to use the equivalent USWDS class here, `.usa-grid`, instead. Let me know if that doesn't work though!